corehq/apps/locations/views.py

```diff
@@ -846,7 +846,10 @@ def post(self, request, *args, **kwargs):
         # We need to start this task after this current request finishes because this
         # request uses the lock_locations decorator which acquires the same lock that
         # the task will try to acquire.
-        task = import_locations_async.apply_async(args=[domain, file_ref.download_id], countdown=10)
+        task = import_locations_async.apply_async(
+            args=[domain, file_ref.download_id, request.couch_user],
```

Comment:
It would be better to just pass the User ID and look it up in Celery rather than pass the entire user object.